services/QuillLMS/config/routes.rb

```diff
@@ -523,9 +523,13 @@
   post '/session/set_post_auth_redirect', to: 'sessions#set_post_auth_redirect'
   resource :session
 
-  resource :account, only: [:new, :create, :edit, :update] do
+  resource :account, only: [:new, :create, :edit, :update, :show] do
     post :role, on: :member
   end
+
+  get 'account/:id/finish_set_up', to: 'accounts#edit'
```

Comment:
Since account is a singular resource, the `:id` here is kind of confusing since it looks like a restful route.  What about using `:token`?

```
  get 'account/:token/finish_set_up', to: 'accounts#edit'
  put 'account/:token', to: 'accounts#update'
```